### ISSUE TITLE
isc: Clean up code for random seeds in resampling tests

### ISSUE THREAD

**Issue by snastase** [brainiak/brainiak#424]
The way we use random seeds in the statistical resampling tests for ISC analyses (e.g., `bootstrap_isc`, `permutation_isc`) is a bit redundant. I think the code was initially written with the `np.random.seed` in mind, then updated to the better `np.random.RandomState` approach—but still has some vestigial fluff. The current version should work fine, but the randomization bit can be slimmed down and made more intuitive. Thanks to @mlnguyen for pointing this out.

**Comment by mihaic** [comment]
@snastase, we are planning on releasing v0.9 soon, as you know. Would you like to delay the release because of this issue? I suggest not delaying, especially because we can have a 0.9.1 release soon afterwards if you want this issue fixed fast in a release.

**Comment by snastase** [comment]
No, I don't think this is worth delaying the release. More of just a stylistic issue—shouldn't negatively affect actual analysis results.

**Comment by emilymicc** [comment]
When I try to run the boostrap analysis, I run into this error
ValueError                                Traceback (most recent call last)
<ipython-input-37-de1a0cd41ab3> in <module>
      4 observed, ci, p, distribution = bootstrap_isc(iscs_floats, pairwise=False,
      5                                               ci_percentile=95,
----> 6                                               summary_statistic='mean')
      7 
      8 # Inspect shape of null distribution

~\Documents\HassonLab\ISC\code\isc_standalone.py in bootstrap_isc(iscs, pairwise, summary_statistic, n_bootstraps, ci_percentile, random_state)
    782 
    783         # Update random state for next iteration
--> 784         random_state = np.random.RandomState(prng.randint(0, MAX_RANDOM_SEED))
    785 
    786     # Convert distribution to numpy array

mtrand.pyx in mtrand.RandomState.randint()

ValueError: high is out of bounds for int32

My computer is an HP Spectrex360 and is a 64-bit machine. When I set MAX_RANDOM_SEED = 2**16-1 instead of 2**32-1, the analysis worked. It appears that anything higher than 2**16-1 creates the same error message.
